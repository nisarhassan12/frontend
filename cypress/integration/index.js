describe('Index page', () => {
  beforeEach(() => {
    cy.log(`Visiting http://localhost:3000`)
    cy.visit('/')
  })

  it('should match index page snapshot', () => {
    cy.get('#__next').toMatchImageSnapshot()
  })

  it('should render logo icon', () => {
    cy.get('a[href="/"] > img').should('have.length', 1)
  })
})
