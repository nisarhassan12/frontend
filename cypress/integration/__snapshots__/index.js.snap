exports[`Index page > should match index page snapshot #0`] = `
<div id="__next">
  <div
    width="1"
    class="sc-bdVaJa sc-bwzfXH NavBar__StyledNav-sc-5bxc2y-1 jTWFKc"
  >
    <a href="/"
      ><img
        height="7vh"
        width="100%"
        src="../../static/logo.svg"
        class="sc-bdVaJa sc-bZQynM ilartJ"
    /></a>
    <div class="NavBar__LinksGroup-sc-5bxc2y-0 blbCBZ">
      <a href="/login">Sign In</a><a href="/register">Sign Up</a>
    </div>
  </div>
  <div class="sc-bdVaJa sc-bwzfXH Primitives__Page-sc-12hrqp0-4 gQrDDx">
    <h1
      font-size="calc(1.8em + 5vw)"
      font-weight="bold"
      class="sc-bdVaJa sc-htpNat sc-bxivhb jcKHLF"
    >
      Komfy
    </h1>
    <h2>Project Status ⚡</h2>
    <p>
      Komfy is a protected social network without annoying trackers and context
      ads. It is currently in active development. User registration will be
      available soon.
    </p>
    <div class="pages__Links-sc-9mrxf5-0 ePUgry">
      <a href="https://github.com/komfy">GitHub 🐙</a
      ><a href="https://twitter.com/KomfySocial">Twitter 🐦</a
      ><a href="https://codecov.io/gh/komfy/frontend">CodeCov 📊</a
      ><a href="https://travis-ci.org/komfy/frontend">Travis CI ⛓</a
      ><a href="https://t.me/komfy">Telegram 📰</a>
    </div>
    <p>
      I'll answer all questions here:&nbsp;<a href="https://t.me/talentless_guy"
        >@talentless_guy</a
      >
    </p>
    <div>{"hello":"world"}</div>
  </div>
</div>
`;
